### DISCUSSION TITLE
Where can I download the receipt for my purhcase??

### DISCUSSION THREAD

**Discussion by EugenBoss** [aclap-dev/video-downloadhelper#1241] [License & Purchase requests]
### Your question

Where can I download the receipt for my purhcase??

### Addon & CoApp version and details

Where can I download the receipt for my purhcase??

**Comment by jcfrog** [comment]
Hi,
You should have received an email from us. Maybe check your spams.
If you don't find it please send us an email at licenses@downloadhelper.net

**Comment by EugenBoss** [comment]
Hello
Where can I download the invoice for my licence???
I did not find it anwyere to download.


Please help

THank you

**Comment by EugenBoss** [comment]
Hello???
